models/base_models.py

```diff
@@ -0,0 +1,18 @@
+from candidate_models.base_models import base_model_pool
+from candidate_models.model_commitments import model_layers
+
+
+def get_model_list():
+    return base_model_pool.keys()
+
+
+def get_model(name):
+    return base_model_pool[name]
+
+
+def get_layers(name):
+    return model_layers[name] if name in model_layers else None
```

Comment:
don't understand the None here. If the `name` is not in `model_layers` this function can't do anything and should throw an error. Or just `return model_layers[name]` which will automatically raise an error if the `name` is not in the dict

Reply:
still needs addressing

Reply:
I'd rather have it that way so that this doesn't break the process! I still want all the other models to be scores when one layers names are not there. It will fail later on either way and throw an exception.

Reply:
That sounds like this kind of error-handling should be in `brainscore/submission`. Otherwise, this code doesn't have a guarantee that `None` doesn't also break the process